server/BraveAlerterConfigurator.js

```diff
@@ -142,7 +166,11 @@ class BraveAlerterConfigurator {
       }
     }
 
-    return session.respondedByPhoneNumber
+    return {
+      respondedByPhoneNumber: session.respondedByPhoneNumber,
+      replaceReturnMessageToRespondedByPhoneNumber,
+      replaceReturnMessageToOtherResponderPhoneNumbers,
```

Comment:
See my suggestion elsewhere about using `replacement` instead of `replace` in these variable names.